src/xo-app/dashboard/overview/index.js

```diff
@@ -148,225 +281,261 @@ class PatchesCard extends Component {
-            </Col>
-            <Col mediumSize={8}>
-              <Card>
-                <CardHeader>
-                  <Icon icon='disk' /> {_('srTopUsageStatePanel')}
-                </CardHeader>
-                <CardBlock className='dashboardItem'>
-                  <BlockLink to='/dashboard/health'>
-                    <ChartistGraph
-                      style={{strokeWidth: '30px'}}
-                      data={{
-                        labels: map(props.topWritableSrs, 'name_label'),
-                        series: map(props.topWritableSrs, sr => (sr.physical_usage / sr.size) * 100)
-                      }}
-                      options={{ showLabel: false, showGrid: false, distributeSeries: true, high: 100 }}
-                      type='Bar'
-                    />
-                  </BlockLink>
-                </CardBlock>
-              </Card>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <PatchesCard hosts={props.hosts} />
-            </Col>
-          </Row>
-        </Container>
-        : <Container><Upgrade place='dashboard' available={3} /></Container>
+                </BlockLink>
+              </div>
+            </CardBlock>
+          </Card>
+        </Col>
+      </Row>
+      <Row>
+        <Col mediumSize={4}>
+          <Card>
+            <CardHeader>
+              <Icon icon='alarm' /> {_('alarmMessage')}
+            </CardHeader>
+            <CardBlock>
+              <p className={styles.bigCardContent}>
+                <Link to='/dashboard/health' className={props.nAlarmMessages > 0 ? 'text-warning' : ''}>{props.nAlarmMessages}</Link>
+              </p>
+            </CardBlock>
+          </Card>
+        </Col>
+        <Col mediumSize={4}>
+          <Card>
+            <CardHeader>
+              <Icon icon='task' /> {_('taskStatePanel')}
+            </CardHeader>
+            <CardBlock>
+              <p className={styles.bigCardContent}>
+                <Link to='/tasks'>{props.nTasks}</Link>
+              </p>
+            </CardBlock>
+          </Card>
+        </Col>
+        <Col mediumSize={4}>
+          <Card>
+            <CardHeader>
+              <Icon icon='user' /> {_('usersStatePanel')}
+            </CardHeader>
+            <CardBlock>
+              <p className={styles.bigCardContent}>
+                {props.isAdmin
+                  ? <Link to='/settings/users'>{nUsers}</Link>
+                  : <p>{nUsers}</p>
+                }
+              </p>
+            </CardBlock>
+          </Card>
+        </Col>
+      </Row>
+      <Row>
+        <Col mediumSize={4}>
+          <Card>
+            <CardHeader>
+              <Icon icon='vm-force-shutdown' /> {_('vmStatePanel')}
+            </CardHeader>
+            <CardBlock className='dashboardItem'>
+              <BlockLink to='/home?t=VM'>
+                <ChartistGraph
+                  data={{
+                    labels: ['Running', 'Halted', 'Other'],
```

Comment:
It would be nice to make the most of this PR and internationalize the labels in the self service view as well. :)